### PR TITLE
Add logs for ignored errors

### PR DESCRIPTION
**Pull request by vmanikes**
*Description of changes:*
I saw that some of the calls such as closing a file, closing a response body return a error, but they are not checked and not even logged. I am adding support to handle that case

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by willarmiros — APPROVED** [review]
LGTM, thanks for the addition @vmanikes!

**Comment by vmanikes** [comment]
Of course @willarmiros

**Comment by vmanikes** [comment]
@willarmiros I am new to this project. When does a PR get merged after approval?

**Comment by willarmiros** [comment]
Right now!

**Comment by vmanikes** [comment]
lol, thanks @willarmiros

**Pull request merged**
